src/extension/css/pom.xml

```diff
@@ -51,6 +51,23 @@
       <artifactId>easymockclassextension</artifactId>
       <scope>test</scope>
     </dependency>
+        <dependency>
+      <groupId>org.geoserver</groupId>
+      <artifactId>gs-rest</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This is odd, I thought we made a test case for css and ysld to confirm this endpoint would work with different style formats :(